unlock-app/src/__tests__/reducers/locksReducer.test.js

```diff
@@ -49,12 +49,15 @@ describe('locks reducer', () => {
     ).toBe(initialState)
   })
 
-  it('should add the lock when receiving CREATE_LOCK and if it was not there yet', () => {
+  it('should add the lock with a nornalized address when receiving CREATE_LOCK and if it was not there yet', () => {
     expect.assertions(1)
     expect(
       reducer(undefined, {
         type: CREATE_LOCK,
-        lock,
+        lock: {
+          ...lock,
+          address: lock.address.toUpperCase(),
```

Comment:
It looks like in the code we normalize by using `.toLowerCase()`, is this difference expected?

Reply:
Yes that's the test ;) I am testing that sending an uppercase address will get normalized into a lowercase...